extern mod std;

#[test]
fn test_hello_world() {
  fail ~"this test has failed because of a reason";
}
